### PR TITLE
Filament cutter

### PR DESCRIPTION
**Pull request by ImChrono**
First version of a Filament Cutter. It is specifically designed for a manual cutter that utilizes a compress pin positioned at a defined X and Y location

### CODE REVIEW

**Review by 3DCoded — CHANGES_REQUESTED** [review]
This looks great so far! I left some comments in `macros.cfg` and `main.cfg`. Aside from those, do you want to [write documentation](https://3dcoded.github.io/3MS/contributing/devsetup/#documentation-changes) for it, or should I write it and check back with you like in your previous PR?

Thank you for efforts to improve the 3MS project.

File: main.cfg
[the remark this answers is not in the document]
Comment (ImChrono) [reply main.cfg:1]: I'm currently working on it. I forgot to comment out before submitting the PR

**Comment by ImChrono** [comment]
> This looks great so far! I left some comments in `macros.cfg` and `main.cfg`. Also, do you want to [write documentation](https://3dcoded.github.io/3MS/contributing/devsetup/#documentation-changes) for it (a separate PR), or should I write it and check back with you like in your previous PR?
> 
> Thank you for efforts to improve the 3MS project.

I can handle the documentation as well, no problem. For now, I can't test it in a CoreXY configuration since my machine doesn't have a manual filament cutter, but it should work as expected.

File: main.cfg
[the remark this answers is not in the document]
Comment (3DCoded) [reply main.cfg:1]: > I'm currently working on it. I forgot to comment out before submitting the PR

Can you please commit the commented lines?

[the remark this answers is not in the document]
Comment (ImChrono) [reply main.cfg:1]: I’ve removed the lines for now: (https://github.com/3DCoded/3MS/pull/21/commits/8e01d92659d320f7e9f38356d83059521f8e8459). It should already be committed.

[the remark this answers is not in the document]
Comment (3DCoded) [reply main.cfg:1]: I meant these lines in `main.cfg`:

```
[include ./cutter/settings.cfg]

...

, 3ms/cutter/macros.cfg
```

[the remark this answers is not in the document]
Comment (ImChrono) [reply main.cfg:1]: I’ve commented out those parts. Now, I'll refine my work and make a separate PR for the RETRACT_BEFORE_CUT macro along with some other tweaks.

**Comment by 3DCoded** [comment]
Thank you for your contribution to the 3MS project!

**Pull request merged**